docs/src/app/(docs)/v7/page.mdx

```diff
@@ -158,14 +158,13 @@ downside, but in order to be non-breaking, could not not be default, and thus
 was not an ideal solution.
 
 In v7, we're moving this option to the server-side route configuration so that
-you have all the configuration for a given file route in one place. We're also
-changing the default behaviour to make waiting opt-in.
+you have all the configuration for a given file route in one place.
 
 ```ts
 // On the client
 await uploadFiles("myUploader", {
   files: [ ... ],
-  // Opt-out of waiting for server data // [!code --]
+  // Opt-out of waiting for server data
```

Comment:
should the [!code --] still be there?

Reply:
there's no diff between pre and post anymore. behavior is the same?